src/Operation/Factory/SubresourceOperationFactory.php

```diff
@@ -127,8 +127,14 @@ private function computeSubresourceOperations(string $resourceClass, array &$tre
                     $operation['operation_name']
                 );
 
+                $prefix = trim(trim($rootResourceMetadata->getAttribute('route_prefix', '')), '/');
```

Comment:
we don't need the first `trim` imo

Reply:
This is lifted straight from here https://github.com/api-platform/core/blob/master/src/Bridge/Symfony/Routing/ApiLoader.php#L204 - do you think I should adjust both or is there another reason why you think both aren't needed. Also would you mind clarifying if you mean the inner or outer trim when you say 'first'?

Reply:
Better keep both, but I think that the inner one is less usefull.
Pretty sure that some users use `"route_prefix"="/foo"` or `"route_prefix"="foo/"` and setting `"route_prefix"="/"`  would be strange but would not have impact at least.